ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/util/EpochProcessorUtil.java

```diff
@@ -259,7 +255,6 @@ public static void process_justification_and_finalization(BeaconState state)
       }
 
     } catch (IllegalArgumentException e) {
-      LOG.log(Level.WARN, e.getMessage());
```

Comment:
Why do we not log at the epoch processor now?

Reply:
Because we throw the exception upwards and it gets logged later.  It was getting logged twice.

As a general rule you either log the exception or throw it but not both.